### ISSUE TITLE
a11y: Voiceover position not resetted when invoked from tab selection

### ISSUE THREAD

**Issue by jantanis** [xamarin/Xamarin.Forms#8848]
### Description
When I switch tabs I want the voiceover to start at the top of the page. I can produce this "reset" function from a button, but it won't work from a tab somehow.

### Steps to Reproduce

1. Open A11y.sln
2. Start iOS project on device (not simulator)
3. Enable voiceover
4. Select button
5. Double tap button -> Voiceover focus gets resetted to top item
6. Switch tab to page 2 and back to page 1 -> Voiceover does not get resetted

### Expected Behavior
When invoking UIAccessibility.PostNotification(UIAccessibilityPostNotification.ScreenChanged, null) voiceover focus gets resetted

### Actual Behavior
Does not work when invoked from tab

### Basic Information

- Version with issue: 4.3.0.991211
- Last known good version: n/a
- IDE: VS for Mac 8.3.10.2
- Platform Target Frameworks: 
  - iOS:  11.2
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices: iPhone with iOS 11

### Screenshots
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/11005949/70615201-43267300-1c0c-11ea-9f68-cd7dc7bcaaf1.gif)

### Reproduction Link
[A11Y.zip](https://github.com/xamarin/Xamarin.Forms/files/3949924/A11Y.zip)

**Comment by PureWeen** [comment]
I'm going to close this one for now

@rachelkang and I played with this one for a bit and the current behavior is all expected from native iOS

Xamarin.Forms itself isn't getting in the way here at all.

I've attached a sample Xamarin.iOS native app where you can see the same behavior
[AcessTest.zip](https://github.com/xamarin/Xamarin.Forms/files/5426920/AcessTest.zip)

If you can elaborate on the type of accessibility requirement you are trying to meet that would help us understand the limitation here and possibly add some additional APIs to help out. If you're trying to select the content after selecting a new tab then you're going to need to work against the default a11y settings of iOS to accomplish this
